apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -246,6 +253,10 @@ def initialize(self):
         self._l3_plugin = None
         self._db_plugin = None
         self.attestator = attestation.EndpointAttestator(self.apic_manager)
+        net_cons_source = cfg.CONF.ml2_cisco_apic.network_constraints_filename
```

Comment:
The requirement here is to be able to modify the constraints and have them take effect without restarting neutron server. Using a regular Neutron config file won't allow that.